site/docs/id/plugins/runner.md

```diff
@@ -77,10 +79,297 @@ run(bot);
+Sebaliknya, ia mengambil update dari `Bot` biasa yang mengontrol worker tersebut.
+
+```asciiart:no-line-numbers
+1. ambil update                                  Bot
+                                              __// \\__
+                                           __/  /   \  \__
+2. kirim update ke workers              __/    /     \    \__
+                                     __/      /       \      \__
+                                    /        /         \        \
+3. proses update            BotWorker   BotWorker   BotWorker   BotWorker
+```
+
+grammY runner menyediakan sebuah middleware untuk mengirim update ke bot workers.
+Bot workers nantinya akan menerima dan menangani update tersebut.
+Dengan begitu, bot utama cuma perlu mengambil dan mendistribusikan update ke semua bot worker yang ditanganinya.
+Penanganan update yang sesungguhnya (mem-filter pesan, membalas pesan, dll) dilakukan oleh bot worker tersebut.
+
+Mari kita lihat bagaimana cara menggunakannya.
+
+### Menggunakan Bot Workers
+
+> Contoh berikut tersedia di [repository grammY runner](https://github.com/grammyjs/runner/tree/main/examples).
+
+Mari kita mulai dengan membuat sebuah instance bot utama untuk mengambil dan mendistribusikan update ke para workers.
+Buat sebuah file bernama `bot.ts` yang berisi kode berikut:
+
+<CodeGroup>
+  <CodeGroupItem title="TypeScript" active>
+
+```ts
+// bot.ts
+import { Bot } from "grammy";
+import { distribute, run } from "@grammyjs/runner";
+
+// Buat bot-nya.
+const bot = new Bot("");
+
+// Opsional, tangani update secara berurutan.
+// bot.use(sequentialize(...))
+
+// Distribusikan update ke bot workers.
+bot.use(distribute(__dirname + "/worker"));
+
+// Jalankan bot secara bersamaan (concurrent) menggunakan multi-threading.
+run(bot);
+```
+
+</CodeGroupItem>
+ <CodeGroupItem title="JavaScript">
+
+```js
+// bot.ts
+const { Bot } = require("grammy");
+const { distribute, run } = require("@grammyjs/runner");
+
+// Buat bot-nya.
+const bot = new Bot("");
+
+// Opsional, tangani update secara berurutan.
+// bot.use(sequentialize(...))
+
+// Distribusikan update ke bot workers.
+bot.use(distribute(__dirname + "/worker"));
+
+// Jalankan bot secara bersamaan (concurrent) menggunakan multi-threading.
+run(bot);
+```
+
+</CodeGroupItem>
+ <CodeGroupItem title="Deno">
+
+```ts
+// bot.ts
+import { Bot } from "https://deno.land/x/grammy/mod.ts";
+import { distribute, run } from "https://deno.land/x/grammy_runner/mod.ts";
+
+// Buat bot-nya.
+const bot = new Bot("");
+
+// Opsional, tangani update secara berurutan.
+// bot.use(sequentialize(...))
+
+// Distribusikan update ke bot workers.
+bot.use(distribute(__dirname + "/worker"));
+
+// Jalankan bot secara bersamaan (concurrent) menggunakan multi-threading.
+run(bot);
+```
+
+</CodeGroupItem>
+</CodeGroup>
+
+Di direktori yang sama dengan `bot.ts`, buat file kedua dengan nama `worker.ts` (seperti yang tertera di baris ke-12 pada kode di atas).
```

Comment:
```suggestion
Pada direktori yang sama dengan `bot.ts`, buat file kedua dengan nama `worker.ts` (seperti yang tertera di baris ke-12 pada kode di atas).
```

Reply:
how? but, it will be strange.

Reply:
Definitely disagree. The correct conjunction is "di", not "pada".

Reply:
it's ok.